pkg/chartvalues/apiextensions_aws_config_e2e.go

```diff
@@ -38,15 +41,24 @@ func NewAPIExtensionsAWSConfigE2E(config APIExtensionsAWSConfigE2EConfig) (strin
 	if config.VersionBundleVersion == "" {
 		return "", microerror.Maskf(invalidConfigError, "%T.VersionBundleVersion must not be empty", config)
 	}
-	if config.AWS.Region == "" {
-		return "", microerror.Maskf(invalidConfigError, "%T.AWS.Region must not be empty", config)
-	}
 	if config.AWS.APIHostedZone == "" {
 		return "", microerror.Maskf(invalidConfigError, "%T.AWS.APIHostedZone must not be empty", config)
 	}
 	if config.AWS.IngressHostedZone == "" {
 		return "", microerror.Maskf(invalidConfigError, "%T.AWS.IngressHostedZone must not be empty", config)
 	}
+	if config.AWS.NetworkCIDR == "" {
```

Comment:
Would it make more sense to default these values instead of returning error? I think if we test `cluster-operator` and install `aws-operator` there, then the values don't matter?

Reply:
I thought about it and I think not really. It makes sense to set those for the CR and host VPC in one place IMO.

Reply:
After some discussions this changed. The subnet configs have to be empty for newer versions. They are optional now and will be removed after some transitioning period.